forms-flow-web/src/helper/helper.js

```diff
@@ -2,4 +2,12 @@ const replaceUrl = (URL, key, value) => {
   return URL.replace(key, value);
 };
 
-export { replaceUrl };
+const getTenantinfo = (process_key, list)=>{
+  const process = list.filter((item)=> item.key === process_key)[0]
+  if(process){
+    return process.tenantKey
```

Comment:
There can be cases here there wont be tenantKey example Default Workflow. Also normal  Environment without Multitenancy

Reply:
for all environments, there will be tenant key property in the list it will be null for normal envs.
![image](https://user-images.githubusercontent.com/93634377/168410548-3d80a8ad-cf6a-41e8-bde8-b075688ae200.png)

Reply:
I need some form of optimization here since filtering is an O(n) operation and this helper method is being called multiple times, either need to avoid these multiple calls or need to implement some kind of memorization.

Reply:
We should avoid Calling multiple times. Had noticed these in the stepper calls.